src/core/stdc/math.d

```diff
@@ -215,65 +215,152 @@ version( CRuntime_DigitalMars )
     }
   }
 }
-else version( CRuntime_Microsoft )
+else version( CRuntime_Microsoft ) // fully supported since MSVCRT 12 (VS 2013) only
 {
+  version( all ) // legacy stuff to be removed in the future
+  {
     enum
     {
-        ///
         _FPCLASS_SNAN = 1,
-        ///
         _FPCLASS_QNAN = 2,
-        ///
         _FPCLASS_NINF = 4,
-        ///
         _FPCLASS_NN   = 8,
-        ///
         _FPCLASS_ND   = 0x10,
-        ///
         _FPCLASS_NZ   = 0x20,
-        ///
         _FPCLASS_PZ   = 0x40,
-        ///
         _FPCLASS_PD   = 0x80,
-        ///
         _FPCLASS_PN   = 0x100,
-        ///
         _FPCLASS_PINF = 0x200,
     }
 
-    ///
+    deprecated("Please use the standard C99 function copysignf() instead.")
```

Comment:
Without a more selective message filtering mechanism I don't think we should output deprecation messages to the VS2010 or VS2012 user. I'd recommend commenting the deprecation attributes.